### PR TITLE
Containerize

### PR DESCRIPTION
**Pull request by andrerom**

### CODE REVIEW

File: dockerfiles/ezphp/Dockerfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ RUN apt-get update -q -y && apt-get install -q -y --force-yes --no-install-recom
     php5-gd \
     php5-curl \
     php5-twig \
+    mysql-client \
```
Comment (andrerom) [inline dockerfiles/ezphp/Dockerfile]: why is this added?

Comment (vidarl) [inline dockerfiles/ezphp/Dockerfile]: In order to be able to import a db dump  using this image/container
Ref. site1initialize in docer-compose_services.yml

**Comment by andrerom** [comment]
+1 for now, merging.
